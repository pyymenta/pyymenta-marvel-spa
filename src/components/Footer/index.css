.footer {
  background: var(--color-primary);
  width: 100%;
  height: 20vh;
}
