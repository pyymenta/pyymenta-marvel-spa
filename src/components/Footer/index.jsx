import React from 'react';
import './index.css';

const Footer = () => <footer className='footer' />;

export default Footer;
