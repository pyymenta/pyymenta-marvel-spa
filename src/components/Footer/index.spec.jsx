import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

describe('Footer component', () => {
  const container = shallow(<Footer />);

  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

  it('should have footer className', () => {
    expect(container.find('.footer').length).toEqual(1);
  });
});
