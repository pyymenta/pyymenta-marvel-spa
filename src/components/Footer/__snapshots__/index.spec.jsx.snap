// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match the snapshot 1`] = `"<footer class=\\"footer\\"></footer>"`;
